pkgs/shells/zsh/antidote/default.nix

```
@@ -0,0 +1,29 @@
+
+{ lib, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  version = "1.8.6";
+  pname = "antidote";
+
+  src = fetchFromGitHub {
+    owner = "mattmc3";
+    repo = "antidote";
+    rev = "v${version}";
+    sha256 = "CcWEXvz1TB6LFu9qvkVB1LJsa68grK16VqjUTiuVG/c=";
```

You could use a SRI hash
```suggestion
    hash = "sha256-CcWEXvz1TB6LFu9qvkVB1LJsa68grK16VqjUTiuVG/c=";

```